src/client/components/mathjax.tsx

```diff
@@ -0,0 +1,7 @@
+// Just use this same config everywhere please
```

Comment:
You'll only see this comment if you happen to find this XD In an ideal world, (1) prolly wrap `MathJaxContext` in a component that uses this fixed `MathJaxConfig` and then (2) have a linter that bans direct usage of `MathJaxContext`; step (2) being significantly harder than (1)

Reply:
MathJax is jank anyway! CJ is making me use KaTeX (which I didn't know about before he made a comment!)

Reply:
back in my day we used `jsMath`! you youngin's and your new toys